lib/Backends/NNPI/NNPI.cpp

```diff
@@ -168,11 +167,14 @@ bool NNPIBackend::isOpSupported(const NodeInfo &NI) const {
     return NI.allInputsAndOutputsHaveSameElemKind(
         {ElemKind::FloatTy, ElemKind::Float16Ty, ElemKind::Int8QTy,
          ElemKind::Int32ITy, ElemKind::Int64ITy});
-
+  case Kinded::Kind::ModuloNodeKind:
+    return NI.allInputsAndOutputsHaveSameElemKind({ElemKind::Int32ITy});
+  case Kinded::Kind::MulNodeKind:
```

Comment:
Was this just incorrectly specified before, i.e. we never had support for `Int32ITy` and `Int64ITy`? Or is this a regression? Noticing we're now blacklisting these tests (e.g. `ArithMul_int32_t`) whereas they weren't before. Also it looks like we're blacklisting other tests already like `ArithMin_int32_t`, which implies to me that their `isOpSupported()` is also incorrectly specified?

Reply:
I'm assuming you are relating to the MulNodeKind, yes, seems that it was incorrectly specified, these tests where passing only in ICE-Ref. 
I did not verified these other operations, as far as I know it int32 and int64 by conversion are required only for indices.

Reply:
Got it -- can you just check to see which of Add/Sub/Max/Min also don't support Index types and update here accordingly?

Reply:
Yes, will do that and update.